src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -344,20 +341,6 @@ private static string GetPackageDirectory(BuildScriptGeneratorContext context)
             return isVirtualEnvPackaged;
         }
 
-        private bool IsCollectStaticEnabled()
```

Comment:
Deleted this because we want components to get these from `options` going forward. This is will ensure us to get hierarchical settings all across the board.